packages/common/src/content/contentUtils.ts

```diff
@@ -22,11 +22,11 @@ import { getFamily } from "./get-family";
 import { parseDatasetId, removeContextFromSlug } from "./slugs";
 import {
   DatasetResource,
-  IGetServiceStatusOptions,
+  IHubContentStatus,
   IHubServiceBackedContentStatus,
 } from "./types";
-import { IFeatureServiceDefinition } from "@esri/arcgis-rest-types";
-import { getService } from "@esri/arcgis-rest-feature-layer";
+import { getService, IGetLayerOptions } from "@esri/arcgis-rest-feature-layer";
+import { isService } from "../resources/_internal/_validate-url-helpers";
```

Comment:
did you want to add a commit to this PR that exposes this as `isServiceUrl()` for use in hub-components?